scripts/test-all.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/local/bin/bash
+
+set -eou pipefail
+
+CFN_LINT_IGNORE="-i W3005"
```

Comment:
Also have a way to create a `.cfnlintrc` file.  If you run cfn-lint from the root folder it will pick up the ignore and you can also add a value for what files to run against.